tests/ashley/machina_extensions/forum/test_forum_view.py

```diff
@@ -364,3 +369,177 @@ def test_testing_query_param(self):
             '<a href="?o=3&whatever=2" class="toggle descending" title="Toggle sorting"></a>',
             html=True,
         )
+
+    def test_testing_topic_is_sticky_stay_sticky_default_order(self):
+        """Request page with a sticky topic and check that the sticky topic is always the first
```

Comment:
We usually put `"""` alone on their lines when the docstring can't fit in a single line. I'm surprised the linter doesn't signal it?

Reply:
no lint was happy, it's fixed and noted for next time